core/assets/bundles/bundle_ja.properties

```
@@ -1339,8 +1431,8 @@ block.alloy-smelter.description = チタンや鉛、シリコン、銅からサ
 block.cryofluid-mixer.description = 水とチタンから冷却に効率的な冷却水を製造します。
 block.blast-mixer.description = 可燃性のピラタイトを石油を使用してさらに爆発性化合物にします。
 block.pyratite-mixer.description = 石炭、鉛、砂から燃えやすいピラタイトを製造します。
-block.melter.description = 石を熱で溶かして溶岩を生成します。
-block.separator.description = 石を水圧で砕き、石に含まれる様々な鉱石を回収します。
+block.melter.description = 石を熱で溶かしてスラグを生成します。
+block.separator.description = 石を水圧で砕き、石に含まれる多様な鉱石を回収します。
```

原文：`Separates slag into its mineral components.`
```suggestion
block.separator.description = スラグを分離して、多様な鉱石を回収します。
```